developer-resources/tools/IntelliJ.md

```diff
@@ -19,6 +19,10 @@ https://www.jetbrains.com/idea/download/index.html
 There is a [tutorial for IntelliJ](../../open-metadata-resources/open-metadata-tutorials/intellij-tutorial) as part of the
 [Egeria Dojo](../../open-metadata-resources/open-metadata-tutorials/egeria-dojo).
 
+Lombok Plugin
+ 
+IntelliJ development environment supports Lombok by default with the current versions (or as an addition plugin with previous versions). You can get more instructions on this [page](https://projectlombok.org/setup/intellij).
```

Comment:
Egeria makes use of Project Lombok. If using JetBrains IntelliJ IDEA ensure the IDEA has the required plugin configured. See https://projectlombok.org/setup/intellij for more information.

That would make the initial paragraph in developer guidelines redundant?